src/Parser.hs

```diff
@@ -0,0 +1,310 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ViewPatterns #-}
+
+module Parser where
```

Comment:
I believe there should be an explicit export list of symbols from the module here, like there is in `Lib`

Reply:
That's right, I'll add one